src/rules/font-family-name-quotes/__tests__/index.js

```diff
@@ -30,6 +30,12 @@ testRule(rule, {
   accept: variablePositiveTests.concat([ {
     code: "a { font-family: \"Lucida Grande\", \"Arial\", sans-serif; }",
   }, {
+    code: "a { font: 1em \"Lucida Grande\", \"Arial\", sans-serif; }",
```

Comment:
Could we get a passing test that shows that a property like `fonts` (with the s on the end) is defo ignored please?